modules/manage/partials/authentication.adoc

```diff
@@ -0,0 +1,1661 @@
+
+ifndef::env-kubernetes[If you <<enable-authentication, enabled authentication with `kafka_enable_authorization=true`>>, you must enable SASL for the Kafka API.]
+
+ifndef::env-kubernetes[]
+In `redpanda.yaml`, enter:
+
+[,yaml,lines=6]
+----
+redpanda:
+  kafka_api:
+    - address: 0.0.0.0
+      port: 9092
+      name: sasl_listener
+      authentication_method: sasl
+----
+endif::[]
+
+==== Enable SASL with TLS encryption
+
+SASL provides authentication, but not encryption. To provide encryption, you can enable TLS in addition to SASL. See xref:{tls-doc}[].
+
+ifdef::env-kubernetes[TLS is enabled in the Helm chart by default.]
+
+ifndef::env-kubernetes[]
+For example, to enable SASL authentication with TLS encryption for the Kafka API, in `redpanda.yaml`, enter:
+
+[,yaml,lines=5-8]
+----
+redpanda:
+  kafka_api:
+    - address: 0.0.0.0
+      port: 9092
+      name: sasl_tls_listener
+      authentication_method: sasl
+  kafka_api_tls:
+    - name: sasl_tls_listener
+      key_file: broker.key
+      cert_file: broker.crt
+      truststore_file: ca.crt
+      enabled: true
+      require_client_auth: false
+----
+endif::[]
+
+[[scram]]
+==== SASL/SCRAM
+
+SASL/SCRAM does not require sending passwords over the network, even in an encrypted form. It uses a challenge-response mechanism, ensuring that the password is not directly accessible to the server. It works with hashed passwords, providing additional security against dictionary attacks.
+
+ifndef::env-kubernetes[]
+===== Enable SASL/SCRAM
+
+SASL/SCRAM is enabled by default.
+
+To check if SASL/SCRAM is enabled:
+
+[,bash]
+----
+rpk cluster config get sasl_mechanisms
+----
+
+You should see `SCRAM` in the output.
+
+If SASL/SCRAM is not enabled, enable it:
+
+[,bash]
+----
+rpk cluster config set sasl_mechanisms '["SCRAM"]'
+----
+endif::[]
+
+===== Create SCRAM users
+
+When you have SASL authentication enabled for your Redpanda cluster, you can create SCRAM users. Redpanda supports the following SASL/SCRAM authentication mechanisms for the Kafka API:
+
+- `SCRAM-SHA-256`
+- `SCRAM-SHA-512`
+
+By default, SCRAM users don't have any permissions in the cluster. Only superusers can grant permissions to new users through ACLs.
+
+. To create the user `myuser` with a password `changethispassword`, run xref:reference:rpk/rpk-acl/rpk-acl-user-create.adoc[`rpk acl user create`]:
++
+```bash
+rpk acl user create myuser \
+  -p 'changethispassword' \
+  --mechanism SCRAM-SHA-256
+```
++
+TIP: Enclose passwords in single quotes to avoid conflicts with special characters. Enclosing characters in single quotes preserves the literal value of each character.
+
+. Use the xref:reference:rpk/rpk-acl/rpk-acl-create.adoc[`rpk acl create`] command to grant `create` and `describe` permissions to `myuser` in the cluster:
```

Comment:
Do we have a page we can link the describes ACLs? E.g., resources (e.g., cluster, topic, group) and permissions (e.g., create, describe).

Reply:
Yes, added.